### ISSUE TITLE
Tests throwing errors

### ISSUE THREAD

**Issue by gRoberts84** [AsperaGmbH/xlsx-reader#18]
Hi there,

I'm currently writing a PR to add a feature but when running tests without any modification to the master code, it's throwing errors:

```
25) Aspera\Spreadsheet\XLSX\Tests\CustomNumberFormatTest::testFormat with data set "scientific notation - exponent larger than 1 digit" ('0.00000000005', '0.00E+0', '5.00E-11')
Undefined array key 0

/xlsx-reader/lib/NumberFormat.php:499
/xlsx-reader/lib/NumberFormat.php:273
/xlsx-reader/lib/NumberFormat.php:189
/xlsx-reader/tests/CustomNumberFormatTest.php:71
```

When debugging the error, it is passing `123.00` as a value to the test, but is then converted to `123.` due to the following code:

```
// Remove insignificant zeroes for now, we will (re-)add them based on format_info next.
if (strpos($number, '.') !== false) {
    $number = rtrim($number, '0');
}
```

This results in this failing, as it returns an empty array:

```
$right_side_chars = str_split($number_parts[1]);
if ($right_side_chars[0] === '') { // Side-effect of str_split('')
    $right_side_chars = array();
}
```

**Comment by rfischer42** [comment]
Greetings, and thank you for the report.

Seems to me like this is related to the recent PHP 8.2 release; It changed the way str_split() works here.
Adding proper PHP 8.2 support is an upcoming task for this library. I presume that's also when this issue will be addressed.

**Comment by rfischer42** [comment]
This issue has been fixed in v1.1.0.
